pkg/pb/overlay.proto

```diff
@@ -20,6 +21,7 @@ message QueryRequest {
     node.Node target = 2;
     int64 limit = 3;
     bool pingback = 4;
+    repeated vouchers.Voucher vouchers = 5;
```

Comment:
why add this to query versus other overlay RPC like Ping or RequestInfo?

Reply:
Also since Satellite won't be on kademlia in the future, it wouldn't also have Nodes service

Reply:
The Query endpoint is where `pingback` is requested, which will put us into either the node's routing table or antechamber